MoltenVK/MoltenVK/Utility/MVKBitArray.h

```diff
@@ -19,6 +19,7 @@
 #pragma once
 
 #include "MVKFoundation.h"
+#include <functional>

```

Comment:
It's mostly a matter of personal taste. `std::function` is used from that header and it's not imported directly nor transitively in `MVKBitArray.h`. That means that every other compilation unit that imports `MVKBitArray.h` must import `functional` before `MVKBitArray.h`, or hope that some other header before `MVKBitArray.h` imports `functional`. This can quickly becomes a bit confusing, so in general I think it's better to import (either directly or transitively) all the headers used in any given file, even if it's a header. But, as I said, it's a matter of personal choice.

Reply:
Fair enough. I'll add it back, in a separate PR.

Reply:
> Fair enough. I'll add it back, in a separate PR.

Included in PR #2382.